com.unity.render-pipelines.high-definition/Editor/PostProcessing/FilmGrainEditor.cs

```diff
@@ -28,7 +28,8 @@ public override void OnInspectorGUI()
 
             if (m_Type.value.intValue == (int)FilmGrainLookup.Custom)
             {
-                PropertyField(m_Texture);
+                using (new IndentLevelScope())
+                    PropertyField(m_Texture);
```

Comment:
You do not need this scope if you are using the attribute, no?

Reply:
We already have the Editor here. I feel the attribute is less instinctive and more hard to read in this case

Reply:
good, I agree :)